posts/entando-personas-make-modular-applications-more-manageable.md

```diff
@@ -0,0 +1,59 @@
+---
+authors:
+- Ford Hatfield
+tags: 
+- Entando
+- Micro-frontends
+- Microservices
+date: 2023-01-12T9:15:00:00.000Z
+title: "Entando Personas Make Modular Applications More Manageable"
+image: ![Multiple people working on a web application](https://github.com/fuhsaz/blog-usa/blob/master/images/2023/01/entndo_personas_blog_image.jpg)
+---
+
+Over the past few years, it has become apparent that businesses need to adapt quickly to changing conditions to stay competitive. Pushing a new feature's time-to-market due to another application part's unpreparedness is unacceptable. Building with a modular architecture in mind can give development teams more independence and potentially improve overall throughput. Unfortunately, modularizing applications can quickly become very complicated and difficult to manage.
```

Comment:
The second sentence sounds a little awkward. Perhaps something like "...due to another part of the application being unprepared is unacceptable"

Reply:
I'll rewrite this and fix the other things you pointed out. Thanks for the review!

Reply:
I agree with Dennis! The second sentence is a little awkward and possibly a bit unclear. I would change the beginning of the sentence to "Delaying the release of a new feature" and then follow up with @dsharpe's amendment.